### PR TITLE
Working sign up, sign in, and logout features

### PR DESCRIPTION
**Pull request by BenSheridanEdwards**

### CODE REVIEW

**Comment by DawidSzpener** [comment]
Looks good

**Review by DawidSzpener — APPROVED** [review]
accepted

File: app/helpers/user_helper.rb [hunk 1]
```diff
@@ -1,2 +1,17 @@
 module UserHelper
+  def signup
+    visit "/signup"
+    fill_in "Name", with: "Ben"
+    fill_in "Email", with: "ben@gmail.com"
```
Comment (even13) [inline app/helpers/user_helper.rb]: in tests, it would be nice to replace @gmail.com with @example.com as Sophie suggested

Comment (BenSheridanEdwards) [reply app/helpers/user_helper.rb]: Will change it to example in the next merge then :D

**Pull request closed without merging**